TASK-27712 Fix mentioning users when editing a post.

mzorai opened the pull request:
The problem comes from the already initialized CKEditor instance, i made sure to reinitialize it when editing a post.

hassenelkamel-exo committed:
TASK-27712: Fix can't mention user when edit a post (#568)

mzorai committed:
MAINT-27712: Make sure to be able to mention users when editing a post.

The problem comes from the already initialized CKEditor instance, i made sure to reinitialize it when editing a post.

ahamdi reviewed (dismissed):
Do not merge until we fix the inifinite loading isue when editing a post

boubaker committed:
TASK-40887 Avoid destroying CKEditor of Activity form on edit (#586)

When editing an activity, the CKEditor is reused and thus, destroying it will cause to re-initialize a new CKEditor.
This PR will improve managing CKEditor instance in Activity Drawer editor
